package.json

```diff
@@ -32,10 +32,8 @@
         "package": "npm run package -w packages/toolkit -w packages/amazonq",
         "newChange": "echo 'Must specify subproject/workspace with -w packages/<subproject>' && false",
         "createRelease": "echo 'Must specify subproject/workspace with -w packages/<subproject>' && false",
-        "format": "prettier --check plugins && npm run format -w packages/ --if-present",
-        "formatfix": "prettier --write plugins && npm run formatfix -w packages/ --if-present",
```

Comment:
Removed all `format` and `formatfix` tasks. They're redundant now that eslint enforces prettier rules.